src/main/webapp/index.html

```diff
@@ -25,6 +25,13 @@
       import { app } from "/scripts/init.js"
 
       function signInStatus() {
+        if (app == null) {
+          return
+        } else if (app.auth().currentUser) {
+          document.getElementById("signed-in").textContent = "Signed in"
+        } else {
+          document.getElementById("signed-in").textContent = "Signed out"
+        }
```

Comment:
could use tertiary a la 
if (app == null) { return }
else { document.gEBI().tC = app.auth().currentUser ? "Signed in" : "Signed out" }

Reply:
Super nifty, thanks!